tests/test_astrometryTask.py

```diff
@@ -211,9 +224,59 @@ def makeSourceCat(self, distortedWcs):
             src.set(sourceCentroidKey, refObj.get(refCentroidKey))
             src.set(sourceInstFluxKey, refObj.get(refFluxRKey))
             src.set(sourceInstFluxErrKey, refObj.get(refFluxRKey)/100)
+
         return sourceCat
 
 
+class TestMagnitudeOutliers(lsst.utils.tests.TestCase):
+    def testMagnitudeOutlierRejection(self):
+        """Test rejection of magnitude outliers."""
```

Comment:
If I'm reading the code correctly, all of these objects live at `(0,0)`, no? Please note in the docstring that this is only testing outlier rejection, not any other part of the matching or astrometry fitter.